upgrade_test.py

```diff
@@ -46,7 +48,14 @@ def truncate_entries(func):
     def inner(self, *args, **kwargs):
         node = args[0]
         with self.db_cluster.cql_connection_patient(node, keyspace='truncate_ks') as session:
-            self.cql_truncate_simple_tables(session=session, rows=NUMBER_OF_ROWS_FOR_TRUNCATE_TEST)
+            InfoEvent(message="Start truncate simple tables").publish()
+            try:
+                self.cql_truncate_simple_tables(session=session, rows=NUMBER_OF_ROWS_FOR_TRUNCATE_TEST)
+            except Exception as details:  # pylint: disable=broad-except
+                InfoEvent(message=f"Failed truncate simple tables. Error: {str(details)}. Traceback: {traceback.format_exc()}",
+                          severity=Severity.ERROR).publish()
+            InfoEvent(message="Finish truncate simple tables").publish()
```

Comment:
move this to inside the try clause, as we need it only if the exception is not raised

Reply:
moved